Assets/Scripts/SimplePhysics.cs

```diff
@@ -130,11 +133,10 @@ void Update()
             mVelY += mGravity;
         }
 
-        // TODO: Bug where character accelerates much more quickly in x dir
         private float UseAccel(float accel, float curSpeed, float maxSpeed)
         {
             float newSpeed = accel + curSpeed;
-            return (newSpeed < maxSpeed) ? newSpeed : maxSpeed;
+            return (Mathf.Abs(newSpeed) < maxSpeed) ? newSpeed : maxSpeed;

```

Comment:
It hasn't appeared to be a problem. Should maxSpeed be compared with an absolute value as well?